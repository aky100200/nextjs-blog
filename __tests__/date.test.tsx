import renderer from 'react-test-renderer'
import React from 'react'

import Date from '../components/date'

test('DateTest', () => {
  const component = renderer.create(<Date dateString='2020-01-01' />)
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
