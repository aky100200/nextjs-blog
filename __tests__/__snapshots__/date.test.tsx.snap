// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DateTest 1`] = `
<time
  dateTime="2020-01-01"
>
  January 1, 2020
</time>
`;
